deployment/docker-osm-healthcare/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM byrnedo/alpine-curl
 MAINTAINER Tim Sutton <tim@kartoza.com>
 
 RUN mkdir /settings
-RUN wget https://planet.openstreetmap.org/pbf/planet-latest.osm.pbf -O settings/country.pbf
+RUN wget https://download.geofabrik.de/asia/indonesia-latest.osm.pbf -O settings/country.pbf
```

Comment:
seems like you unintentionally committed this 😬

Reply:
it's from previous commit :grimacing: